React/exercises/components-worksheet.md

```diff
@@ -0,0 +1,25 @@
+# React Components Worksheet
+
+## Part 1
+
+1. Create an **Avatar** component which takes an image URL and presents that image in a circular shape, with any CSS adjustments you would like (e.g. a ring/border around the photo, or a color filter or gradient).
+1. Create a **Badge** component which takes a number and displays that number in a colored circle or lozenge shape. If the number is less than 10, the background of the badge should be red, if it’s between 10 and 49 it should be yellow, and if 50+ it should be green.
+1. Create a **LineMeter** component, which takes a percentage (a floating point number between 0.0 and 1.0, with 1.0 meaning 100%) and displays an “empty” horizontal bar of a fixed width, and on top of that an additional “filled” horizontal bar the length of which is based on the percentage.
```

Comment:
Overall, I feel like creating all of these components and relating them together might be pretty  challenging. Could we remove this one because the others are used in Part 2 below?